src/App/App.js

```diff
@@ -36,8 +33,24 @@ class App extends Component {
     this.setState({ ...this.state, isMovieView: false });
   }
  viewMovie = (id) => {
-    console.log('click')
-    this.setState({...this.state, isMovieView: true, movie:{...this.state.movie, id: id}});
+    getMovie(id)
```

Comment:
Passing the `id` as the endpoint ended up being the move vs updating the id. When the `id` was updated it reloaded the components and messed up the process.